Gulpfile.js

```diff
@@ -144,3 +146,21 @@ gulp.task('es6.full', function() {
         'ECMASCRIPT_2017')))
     .pipe(gulp.dest('dist/es6'));
 });
+
+gulp.task('spec', function() {
+  return bikeshed('spec/index.bs', 'dist/spec/index.bs.html');
+});
+
+gulp.task('spec.watch', function() {
+  return gulp.watch(
+      'spec/**\/*.bs',
+      ({ path: bspath }) => {
+          const reldir = path.relative(__dirname, path.dirname(bspath));
+          const outfile = path.join(
+              __dirname,
+              'dist',
+              reldir,
+              `${ path.basename(bspath) }.html`);
```

Comment:
Could you do `${ path.basename(bspath, '.html') }.html` instead? It might be easier to link to if the output file is `index.html`.

Reply:
Changed to `${ path.basename(bspath, '.bs') }.html`